RiotSharp/Interfaces/IRiotApi.cs

```diff
@@ -145,38 +145,36 @@ public interface IRiotApi
 
         #region League
         /// <summary>
-        /// Retrieves the league entries for the specified summoners, you can submit more than 10 summoner ids.
+        /// Retrieves the leagues for the specified summoner.
         /// </summary>
-        /// <param name="region">Region in which you wish to look for the leagues of summoners.</param>
-        /// <param name="summonerIds">The summoner ids, not limited to 10.</param>
-        /// <returns>A map of list of league entries indexed by the summoner id.</returns>
-        Dictionary<long, List<League>> GetLeagues(Region region, List<long> summonerIds);
+        /// <param name="region">Region in which you wish to look for the leagues of the summoner.</param>
+        /// <param name="summonerId">The summoner id.</param>
+        /// <returns>A list of leagues.</returns>
```

Comment:
Add cref here

Reply:
Something like this : <see cref="Region"/> and <see cref="summonerId"/> ?

Reply:
I'm something like this `/// <returns>A list of <see cref="League" />.</returns>`

Reply:
Oh yeah that's what i wanted to say but i had forgotten to put the code between \`